pkg/rest/retry.go

```diff
@@ -73,7 +75,7 @@ func GetWithRetry(client *http.Client, url string, apiToken string, settings Ret
 	} else {
 		additionalMessage := ""
 		if resp.StatusCode == 403 {
-			concurrentDownloadLimit := ConcurrentRequestLimitFromEnv(false)
+			concurrentDownloadLimit := environment.GetEnvValueInt(environment.ConcurrentRequestsEnvKey)
```

Comment:
As discussed, do you think this could move into download? 
On any other (non-parallel) use of GetWithRetry - e.g. during deployment this wrongly suggest that a 403 could mean rate-limiting, and changing the env var would actually not make any difference at all

Reply:
Change Done
I still think this change was too expensive compared to the risk you are mentioning
I hope that, in the future, I will not have to make the choice of not helping users with their errors to avoid delaying my project.